runtime/integration-tests/src/env.rs

```diff
@@ -255,7 +255,7 @@ pub mod utils {
 			frame_system::CheckNonce::<T>::from(nonce),
 			frame_system::CheckWeight::<T>::new(),
 			pallet_transaction_payment::ChargeTransactionPayment::<T>::from(0),
-			frame_metadata_hash_extension::CheckMetadataHash::<T>::new(false),
+			frame_metadata_hash_extension::CheckMetadataHash::<T>::new(true),
```

Comment:
If we have this `false`, then IT tests work?

Reply:
Yes because it disables the metadata hash check.

Reply:
I think we can skip checking the metadata for IT. What knowledge gives us IT if that value is `true`?

Reply:
I wanted to make the same proposal. In the end, we expect the SDK to ensure the validity of this API.

Reply:
For signed-related things, we also mock the nonce and similar things for IT, so I think we're fine not checking that. It feels like out of scope (?)